pnp/PnpHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace PnpHelpers
+{
+    public static class PnpHelper
+    {
+        public const string TelemetryComponentPropertyName = "$.sub";
+        private const string PropertyComponentIdentifier = "\"__t\": \"c\",";
+
+        public static string CreateReportedPropertiesPatch(string propertyName, object propertyValue, string componentName = null)
```

Comment:
I would love to see createTelemetry message to be on same lines. Can we use Bytes API from device client so that helper does the needed pnp stuff on properties/headers?

Reply:
I agree, I'll see what I can add without having to bring in an existing nuget dependency